packages/dds/tree/src/simple-tree/treeApi.ts

```diff
@@ -311,8 +311,10 @@ export interface TreeChangeEvents {
 
 	/**
 	 * Emitted by a node when something _may_ have changed anywhere in the subtree rooted at it.
-	 *
 	 * @remarks
+	 * Emitted after any `nodeChanged` events for the change and any `treeChanged` events
```

Comment:
What do you think of:
"This event 'bubbles up', that is, it is emitted after any `nodeChange` events for..."

Bubbling is a familiar JS concept that will help give readers an intuition of the behavior before they read the precise details (which I had to read three times before I got it :D).

Reply:
+1

Reply:
I think that implies that we would trigger the sequence of events bubbling up the tree after each change, and we do not do that. We do need a better description that "after". I guess sometime before the current JS turn yields, afterBatch will be triggered, and it will happen before that.

Reply:
descendants (not dependents)
LGTM